src/test/java/de/retest/recheck/configuration/ProjectConfigurationTest.java

```diff
@@ -0,0 +1,28 @@
+package de.retest.recheck.configuration;
+
+import static de.retest.recheck.configuration.ProjectRootFinderUtil.getProjectRoot;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Test;
+
+class ProjectConfigurationTest {
+
+	ProjectConfiguration cut = ProjectConfiguration.getInstance();
+
+	@Test
+	void local_config_file_should_be_created() throws Exception {
+		cut.ensureProjectConfigurationInitialized();
+
+		final Path projectConfigFolder = getProjectRoot().resolve( ProjectConfiguration.RETEST_PROJECT_CONFIG_FOLDER );
+		final Path projectConfigProperties = projectConfigFolder.resolve( ProjectConfiguration.RETEST_PROJECT_PROPERTIES );
+
+		assertThat( projectConfigFolder.toFile().exists() ).isTrue();
```

Comment:
You can directly to stuff like `assertThat( path ).exists()` or `assertThat( path ).isDirectory`.